mmv1/products/cloudidentity/Group.yaml

```diff
@@ -63,9 +74,9 @@ parameters:
       [API reference](https://cloud.google.com/identity/docs/reference/rest/v1beta1/groups/create#initialgroupconfig)
       for possible values.
     values:
-      - 'INITIAL_GROUP_CONFIG_UNSPECIFIED'
-      - 'WITH_INITIAL_OWNER'
-      - 'EMPTY'
+      - :INITIAL_GROUP_CONFIG_UNSPECIFIED
+      - :WITH_INITIAL_OWNER
+      - :EMPTY
     default_value: :EMPTY
     url_param_only: true
     immutable: true
```

Comment:
Agh. Seems my understanding of how the enum + default_value work is incorrect. I guess I need to explicitly set `initial_group_config` in the example?

Reply:
The default will be set if no other value is set for the field. You have it set to a different value than EMPTY: https://github.com/GoogleCloudPlatform/magic-modules/pull/7321/files#diff-a95af37c581e08b09558b874de691ac0f21fea6108b4f8716a3b1e07370059cbR209

Your last commit removed it from the documentation example but not the test because `skip_test` and the test itself is handwritten!

Reply:
Woops. 🤦 Thank you!